doc/release-notes/8160-thumbnail-limit.md

```diff
@@ -0,0 +1 @@
+New defaults have been added for when to create thumbnails for images and PDFS. The default is 3 MB for images, 1 MB for PDFs. Previously, there was no default.
```

Comment:
Thanks for the release note, @pdurbin!

Reply:
@djbrooke sure, please feel free to edit, of course, even if only to change PDFS to PDFs. 😄